package.json

```diff
@@ -22,19 +22,17 @@
   "dependencies": {
     "cool-ascii-faces": "1.3.3",
     "datejs": "1.0.0-rc3",
-    "github": "0.2.3",
-    "googlemaps": "0.1.20",
-    "hipchatter": "~0.1",
-    "hubot": "2.11.0",
-    "hubot-cron": "0.2.7",
-    "hubot-hipchat": "2.7.5",
-    "hubot-scripts": "2.5.16",
-    "moment": "2.9.0",
+    "github": "0.2.4",
+    "hubot": "2.13.2",
+    "hubot-cron": "0.2.9",
+    "hubot-hipchat": "2.12.0-5",
+    "hubot-scripts": "2.16.1",
+    "moment": "2.10.3",
```

Comment:
used in [`coffeepot.coffee`](https://github.com/detroit-labs/hubot-badger/blob/master/scripts/coffeepot.coffee)